content/en/blog/introduction_to_wamr_running_modes/index.md

```diff
@@ -0,0 +1,185 @@
+
+## Why? Why so many running modes?
+
+Our goal is to provide user the adaptability and flexibility. User could choose one to achieve the best performance and resource usage according to deploy environment and nature of application. But as more running mode supported over the years, the learning curve for new user now is a little unfriendly. But let's go through this blog together, I believe when you finish reading it, you would understand our intention to develop each running mode, and feel confident to choose the one (or combination) you need.
+
+## General introduction of each running mode
+
+WAMR support six running modes, AOT, Classic Interpreter, Fast Interpreter, LLVM JIT, Fast JIT, Multi-tier JIT. Based on core concept and implementation behind, it could be roughly divided into three categories: AOT, Interpreter, JIT
+
+- AOT: Ahead-of-Time compilation. As you can guess from the name, we first need to use the **wamrc compiler to compile wasm file to the AOT file**. Then it could be run with our iwasm vmcore. In this running mode, we could achieve the nearly native speed(the **fastest** of all running modes) with very **small footprint** and **quick startup**
+It could run wasm applications in Interpreter/JIT running mode:
+
+- Interpreter: Interpreters are very useful when **debugging or studying**, but their **performance is relatively poor** compared with other running modes. We support two running modes of the interpreter:
+  - Classic Interpreter: plain interpreter running mode
+  - Fast Interpreter: as you can guess from the name, the fast interpreter runs ~2X faster than the classic interpreter but consumes about 2X memory to hold the pre-compiled code.
+
+- JIT: Using the **Just-in-Time** compilation technique, we could make iwasm run much **faster than Interpreter** mode and sometimes very close to the speed of AOT running mode. We support two running modes of JIT:
+  - LLVM JIT: the JIT engine is implemented based on LLVM codegen. The **performance** of LLVM JIT is **better** than Fast JIT, with ~2x of the latter. But the **startup time** is **slower** than Fast JIT.
+  - Fast JIT: the JIT engine is implemented based on self-implemented codegen and asmjit encoder library. It is a lightweight JIT engine with small footprint, **quick startup**, good portability and relatively good performance. Currently it supports x86-64 target and Linux/Linux-SGX/MacOS platforms. The **performance** of Fast JIT is **~50%** of the performance of LLVM JIT.
+  - Multi-tier JIT: Multi-tier JIT: take advantage of both Fast JIT and LLVM JIT. Enable auto tier up from Fast JIT to LLVM JIT: first run wasm in Fast JIT, LLVM JIT threads run on background gradually replace Fast JIT function pointer, so after a short while later, if wasm run again it will be in LLVM JIT. It provides **both** a **quick startup** and a **decent performance**. Overall performance is excellent and serves as a rather nice trade-off compared with Fast JIT and LLVM JIT.
```

Comment:
`if wasm run again it will be in LLVM JIT`: not precise, iwasm isn't running again, but just gradually switching to LLVM JIT mode
How about:
`so after a short while, iwasm gradually switches into LLVM JIT mode.`